### ISSUE TITLE
Error in diamond makedb related to taxonmap file 

### ISSUE THREAD

**Issue by matrs** [qiyunlab/HGTector#76]
Hello, 
I've followed the tutorial to build a diamond database `hgtector 2.0b2`:

```
taxon_map="hgtector_db/taxon.map.gz
echo $'accession\taccession.version\ttaxid' > prot.accession2taxid
zcat $taxon_map | awk -v OFS='\t' '{split($1, a, "."); print a[1], $1, $2}' >> prot.accession2taxid

diamond makedb --threads 24 --in  db.faa --taxonmap prot.accession2taxid \
--taxonnodes taxdump/nodes.dmp --taxonnames taxdump/names.dmp --db diamond/db
```

and I found this error at the end of the process (after `diamond` has already created a 30GB plus file):

```
Loading sequences...  [2.659s]
Masking sequences...  [3.314s]
Writing sequences...  [0.993s]
Writing accessions...  [1.747s]
Hashing sequences...  [0.21s]
Loading sequences...  [0s]
Writing trailer...  [2.748s]
Loading taxonomy...  [0.258s]
Error: Tokenizer Exception
Error: Tokenizer Exception
```

I didn't find much information about this, so I tried a few things and finally using the `prot.accession2taxid.gz` (6.1GB) from the NCBI solved the problem.

The original `prot.accession2taxid` (created following the instructions) is 500MB gzipped  (still can share it if you want). If that's too much, maybe some hints about what could be causing this problem will be helpful. The file looks OK to me, so I'm not sure to  what to look at.

Cheers!

**Comment by zakisaati** [comment]
I had the same problem, but I solved it with a specific version of Diamond. I tryed with many different versions and the version 0.9.26 was the only one that worked properly in your steps

**Comment by matrs** [comment]
Thanks for letting me know @zakisaati .

**Comment by matrs** [comment]
I tried the last version of `diamond` prior v2, `0.9.36`, and  it worked. Maybe that information can be added to the `README`

**Comment by matrs** [comment]
I tried the last version of `diamond` prior v2, `0.9.36`, and  it worked. Maybe that information can be added to the `README`

**Comment by qiyunzhu** [comment]
This issue was eventually solved in #97 . Sorry for being so late... Now any version of DIAMOND should work.
